src/openms/source/ANALYSIS/ID/FalseDiscoveryRate.cpp

```diff
@@ -625,7 +626,7 @@ namespace OpenMS
     {
       vector<IdentificationData::QueryMatchRef> best_matches =
         id_data.getBestMatchPerQuery(score_ref);
-      for (auto match_ref : best_matches)
+      for (auto match_ref : best_matches) // NOTE: performs copy, should not be necessary?
```

Comment:
@cbielow looking at `handleQueryMatch_` I think that `match_ref` should actually be `const` and that we should not have a copy here. what do you think?